docs/contributor/README.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Onboarding
+This document serves as a prerequisite knowledge checklist for Open Source contributors (Contributor(s)) to the Geoconnex system (System). The System is composed of multiple components which in turn leverage various Open Source software projects and their toolchains.
+
+The system works as follows:
+
+![pid-server](/docs/img/pid-server.png)
+
+## Topics
+For specific contributor documentation on a particular software, follow the contributor spepcific documentation:
+- [Sitemap Generator](./sitemap-generator/)
+- [Yourls Action](./yourls-action/)
+
+
+# Prerequisite Knowledge
+
+The Contributor should become proficient with the following programming languages, data models, file formats, toolchains, and last but not least, the Geoconnex Components and related dependent services that comprise the System.
+
+## Programming Languages
+
+### [Python](https://python.org)
+What the [Yourls Action](https://github.com/cgs-earth/yourls-action) and [Sitemap Generator](https://github.com/cgs-earth/sitemap-generator) for the Geoconnex Permanent Identifier (PID) Service are written in.
+
+### [PHP](https://www.php.net/)
+What the [PID Service](#pid-server) Geoconnex component is written in.
+
+### [Bash](https://www.gnu.org/software/bash/manual/bash.html)
+Used to create various scripts in the [Geoconnex component](#geoconnex-components) toolchain
```

Comment:
Very nitpicky here but the other lines have periods at the ends; this one doesn't

Reply:
nitpick is why you review :)